### PR TITLE
adjusted farmerbot with test net details

### PR DESCRIPTION
**Pull request by Mik-TF**
Added farmerbot info for test net

also added scott's video on the farmerbot

updated get started TOC to include new ssh connection section

@xmonader

### CODE REVIEW

**Comment by Mik-TF** [comment]
@brandonpille 
@scottyeager

Please let me know if something is off. In that case, I can update the file.

I followed your replies (yours and Scott's) [here](https://forum.threefold.io/t/farmerbot-basic-guide-for-qa-net/3845/14).

File: src/farmerbot/farmerbot.md [hunk 1]
```diff
@@ -29,12 +37,12 @@ The key feature of the farmerbot is powermanagement. The farmerbot will automati
 There are 3 main steps to run the Farmerbot on your 3node Threefold farm.
 
 * Prepare Your Farm for the Farmerbot with WOL
-* Move Your Farm to QA Net
+* Move Your Farm to Test Net (or QA Net)
 * Set the Farmerbot
 
-Note that this guide is for QA Net. This is a testing environment and farmers do not receive farming rewards (TFT) when deploying on this network. We invite farmers to try this new feature if they want to help test the farmerbot. We will announce when the farmerbot is ready for the Test Net and the Main Net, where farming rewards are distributed.
+The farmerbot is currently available for Test Net and QA Net.
```
Comment (brandonpille) [inline src/farmerbot/farmerbot.md:43]: Just to be precise for dev net also

File: src/farmerbot/farmerbot.md [hunk 2]
```diff
@@ -83,29 +91,19 @@ Your farmerbot can be run on any system, including on a node. It doesn't have to
...
+If you are on Main Net and you want to try the farmerbot, you should move your farm to either Test Net or QA Net.
 
-To move your farm to QA Net, you simply need to create a new bootstrap image for the QA Net instead of the Test or Main Net.
+To move your farm to a different network, you need to create a new bootstrap image for the new network instead of your current network. You should also wipe your 3nodes' disks before moving to a different network.
 
-To download the QA Net version of the bootstrap, go to the usual bootstrap link [https://v3.bootstrap.grid.tf/](https://v3.bootstrap.grid.tf/) and select QA net.
+To download the Zero-OS bootstrap image, go to the usual bootstrap link [https://v3.bootstrap.grid.tf/](https://v3.bootstrap.grid.tf/) and select the network you want.
 
-![qa_net|690x422](img/farmerbot_4.png) 
+Note that the farmerbot is currently available on QA Net and Test Net.
```
Comment (brandonpille) [inline src/farmerbot/farmerbot.md:102]: Dev net too.

**Review by brandonpille — APPROVED** [review]
Just one small comment. Already approving.

File: src/farmerbot/farmerbot.md (hunk 1 above)
Comment (Mik-TF) [reply src/farmerbot/farmerbot.md:43]: Will change this asap

**Comment by Mik-TF** [comment]
@brandonpille 
Done here!

I adjusted the texts in many places to take into account Dev net, as you suggested.

I also added some information and link to the new SSH connection guide. This will help farmers if they want to deploy the farmerbot on a full VM.

I also wrote how much tft is should cost.

**Pull request merged**